assets/styles/style.css

```diff
@@ -1,35 +1,42 @@
+html {
+  scroll-behavior: smooth;
+}
+
 body {
   min-width: 290px;
   color: rgb(77 78 83);
   background-color: rgb(255 255 255);
   font-family: "Times New Roman", Times, serif;
 }
 
-header {
-  color: black;
-  text-align: center;
-  font-size: 1.9em;
-  margin: 15px;
-}
 #navbar {
   position: fixed;
   min-width: 290px;
-  top: 0px;
-  left: 0px;
+  top: 0;
+  left: 0;
   width: 300px;
   height: 100%;
   border-right: solid rgba(0 22 22 40%);
   border-color: rgba(0 22 22 40%);
+  overflow: auto;
+}
```

Comment:
Page presentation good and alike to the original. Reduce the thickness of your nav-link and make them look good